loopchain/blockchain/blocks/v1_0/block_builder.py

```diff
@@ -56,18 +57,15 @@ def prev_receipts(self, prev_receipts):
             receipt.pop("blockHash", None)
 
     def reset_cache(self):
-        # TODO In 'LFT', this method call is unnecessary. Delete after confirmation.
-        super().reset_cache()
-
+        self.block = None
+        self.hash = None
```

Comment:
Can you explain why you removed `super().reset_cache()`, then add two lines?

Reply:
The purpose disappeared now to change codes, it should be recovered to the past.

Reply:
Sugguest to remove `reset_cache`.